ElectronClient/gui/ConfigScreen.jsx

```diff
@@ -279,8 +292,26 @@ class ConfigScreenComponent extends React.Component {
 			borderRadius: 4,
 		});
 
+		const detectTheme = (autoDetect) => {
+			this.setState({ shouldUseDarkColors: bridge().shouldUseDarkColors() });
+
+			let changeSetting = '';
+			// active theme should change to preferred theme based on system detected theme
+			if (autoDetect) {
```

Comment:
I'll remove this if() statement as it's not needed now. It was there because of previous spec in which `Preferred Light/Dark` options were always visible